lib/ToolbarPanelHub.jsm

```diff
@@ -190,6 +193,16 @@ class _ToolbarPanelHub {
     });
   }
 
+  removeMessages(win, containerId) {
+    const doc = win.document;
+    const messageNodes = doc
+      .getElementById(containerId)
+      .querySelectorAll(".whatsNew-message");
```

Comment:
nit: Might help to pass class selector '.whatsNew-message' here so that this method is truly generic as name suggest 'removeMessages' or consider renaming it to removeWhatsNewMessages, wdyt?

Reply:
Well, `renderMessages` also has hardcoded `.whatsNew-message` and this .jsm handling panel messages only supports whatsnew messages, so I guess this is fine for now. We might support non "whatsnew" messages in the panel later, but we still probably want to remove all messages (and maybe rename the classname at that point).

Reply:
Sounds good, lets get it in master, hitting merge